src/vitest.ts

```diff
@@ -0,0 +1,11 @@
+import { DefaultReporter } from 'vitest/reporters';
```

Comment:
I see that Vitest also has a `BasicReporter`. How does the output from that differ from what we're doing here?

Reply:
The basic reporter is the same as the default reporter, but with the summary disabled. Basically the opposite of what we want.

Reply:
Sorry, my bad. The basic reporter seems to be the same we want, except that it does not print the progress in real time, only at the end.

That makes me think that the summary is not what I have been calling "Summary", but the output from every file. Perhaps we should find a better name for our reporter?

https://github.com/user-attachments/assets/e10011d2-5fa4-431d-b2a6-f0e4a6b21043

Reply:
By the discussion here, it seems what we want is in fact the summary, so `SummaryReporter` sounds like a good name https://github.com/vitest-dev/vitest/issues/7881